tailwind.config.js

```diff
@@ -2,12 +2,37 @@ const { nextui } = require("@nextui-org/react");
 
 /** @type {import('tailwindcss').Config} */
 export default {
-  content: [
-    "./node_modules/@nextui-org/theme/dist/**/*.{js,ts,jsx,tsx}"
-  ],
-  theme: {
-    extend: {},
-  },
-  darkMode: "class",
-  plugins: [nextui()],
+    content: [
+        "./src/**/*.{js,ts,jsx,tsx}",
+        "./node_modules/@nextui-org/theme/dist/**/*.{js,ts,jsx,tsx}"
+    ],
+    theme: {
+        extend: {},
+        screens: {
+            'sm': '640px',
+            'md': '768px',
+            'lg': '1024px',
+            'xl': '1280px',
+            '2xl': '1536px',
+        }
+    },
+    darkMode: "class",
+    plugins: [nextui({
+        layout: {
+            dividerWeight: "1px", // h-divider the default height applied to the divider component
+            disabledOpacity: 0.5, // this value is applied as opacity-[value] when the component is disabled
+            fontSize: {
+                tiny: "8pt", // text-tiny
+                small: "10.5pt", // text-small
+                medium: "1rem", // text-medium
```

Comment:
This one is using a different unit from the rest. Intentional?

Reply:
I just haven't decided what medium should be yet.